### PR TITLE
chore(3iD): Use metamask test wallet from GHA secrets to to test sdk/claims

### PR DESCRIPTION
**Pull request by tangrammer**
# Description

Relates 
https://github.com/kubelt/kubelt/pull/566#issuecomment-1187778097
and 
https://github.com/kubelt/kubelt/pull/566#issuecomment-1189233497

## Type of change

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- [x] [com.kubelt.sdk.v1.oort-test /claims-test](https://github.com/kubelt/kubelt/blob/claims-test/src/test/com/kubelt/sdk/v1/oort_test.cljs#L21)

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation website
- [X] I have made corresponding changes to the literal docs
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/test/com/kubelt/lib/test_utils.cljs [hunk 1]
```diff
@@ -19,7 +19,16 @@
         w)
       (catch js/Error err (do (is false err) err)))))
 
-(defn delete-wallet [app-name wallet-name wallet-password]
+(defn import-wallet [app-name wallet-name mnemonic wallet-password]
+  (go
+    (log/debug "importing-wallet: " wallet-name)
+    (try
+      (let [{:keys [wallet/name]} (<p! (wallet/import& app-name wallet-name mnemonic wallet-password))]
+        (is true (println (str "wallet imported: " name)))
```
[the remark this answers is not in the document]
Comment (tangrammer) [reply src/test/com/kubelt/lib/test_utils.cljs]: done!

File: src/test/com/kubelt/rpc/test_commons.cljs [hunk 2]
```diff
@@ -1,11 +1,15 @@
 (ns com.kubelt.rpc.test-commons
   (:require
-   [com.kubelt.lib.util :as lib.util :refer [node-env]]))
+   [com.kubelt.lib.util :as lib.util :refer [node-env environment]]))
```
[the remark this answers is not in the document]
Comment (tangrammer) [reply src/test/com/kubelt/rpc/test_commons.cljs:3]: yes! moved to proper test folder so it runs in node context

File: src/test/com/kubelt/rpc/test_commons.cljs [hunk 3]
```diff
@@ -1,11 +1,15 @@
...
 
 (def app-name "com.kubelt.ddt.js")
 (def wallet-name "auth_test")
 (def wallet-password "auth_foo_pw")
 
+(def test-wallet-name "test_metamask")
+(def test-wallet-password (get (environment) "METAMASK_PASSWORD"))
```
Comment (crimeminister) [inline src/test/com/kubelt/rpc/test_commons.cljs:10]: Can we skip tests locally for which the necessary environment variables are not set?

**Review by crimeminister — APPROVED** [review]

File: src/test/com/kubelt/rpc/test_commons.cljs (hunk 3 above)
Comment (tangrammer) [reply src/test/com/kubelt/rpc/test_commons.cljs:10]: sounds good!, I'll set up this feature in another PR

Comment (tangrammer) [reply src/test/com/kubelt/rpc/test_commons.cljs:10]: https://github.com/kubelt/kubelt/issues/585

**Pull request merged**